src/Avalonia.Controls/Viewbox.cs

```diff
@@ -129,5 +143,28 @@ protected override Size ArrangeOverride(Size finalSize)
 
             return finalSize;
         }
+
+        private class ViewboxContainer : Control
```

Comment:
Would be good IMO to document this control and why it's needed directly in code.

Reply:
Thanks!